src/test/java/tests/SearchTest.java

```diff
@@ -0,0 +1,48 @@
+package tests;
+
+import base.AbstractBaseTest;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import pages.HomePage;
+import pages.SpongeBobCollectionPage;
+
+public class SearchTest extends AbstractBaseTest {
+
+    @Test()
+    public void checkSearch() throws InterruptedException {
+        HomePage homePage = new HomePage(driver);
+        SoftAssert softAssert = new SoftAssert();
+        Actions actions = new Actions(driver);
+        SpongeBobCollectionPage productPage = new SpongeBobCollectionPage(driver);
+
+        homePage.
+                clickSearchField()
+                .searchInputField().sendKeys("PUMA x SPONGEBOB");
+
+        homePage.
+                clickClearBtn();
+
+        softAssert.assertTrue(homePage.elementIsNotPresent(), "element1");
```

Comment:
what is element1? please reconsider naming

Reply:
changed to inputFieldIsNotPresent